src/main/java/org/darisadesigns/polyglotlina/ManagersCollections/GrammarManager.java

```diff
@@ -40,8 +41,10 @@
     private final List<GrammarChapNode> chapters;
     private final Map<Integer, byte[]> soundMap;
     private GrammarChapNode buffer;
+    public DictCore core;
```

Comment:
I know it's old fashioned, but just for the sake of consistency, I do like to use getters and setters in the PolyGlot project.

Reply:
Fixed